### PR TITLE
Initial project setup

### PR DESCRIPTION
**Pull request by vexonius**
Initiated new project with UI changes shown below

Main view screen

### CODE REVIEW

File: QuizApp/AppDelegate.swift [hunk 1]
```diff
@@ -0,0 +1,33 @@
+//
```
Comment (istaudacher) [inline QuizApp/AppDelegate.swift]: Please remove all comments

File: QuizApp/AppDelegate.swift [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+import UIKit
+
+@main
+class AppDelegate: UIResponder, UIApplicationDelegate {
+
+    func application(_ application: UIApplication, didFinishLaunchingWithOptions launchOptions: [UIApplication.LaunchOptionsKey: Any]?) -> Bool {
+        // Override point for customization after application launch.
+        return true
```
Comment (istaudacher) [inline QuizApp/AppDelegate.swift]: return is not needed

File: QuizApp/AppDelegate.swift [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+    }
+
+    // MARK: UISceneSession Lifecycle
+
+    func application(_ application: UIApplication, configurationForConnecting connectingSceneSession: UISceneSession, options: UIScene.ConnectionOptions) -> UISceneConfiguration {
+        // Called when a new scene session is being created.
+        // Use this method to select a configuration to create the new scene with.
+        return UISceneConfiguration(name: "Default Configuration", sessionRole: connectingSceneSession.role)
```
Comment (istaudacher) [inline QuizApp/AppDelegate.swift]: as above

File: QuizApp/SceneDelegate.swift [hunk 4]
```diff
@@ -0,0 +1,54 @@
+//
```
Comment (istaudacher) [inline QuizApp/SceneDelegate.swift]: Please remove all comments.

File: QuizApp/ViewController.swift [hunk 5]
```diff
@@ -0,0 +1,20 @@
+//
```
Comment (istaudacher) [inline QuizApp/ViewController.swift]: Please remove all comments.

**Review by istaudacher — CHANGES_REQUESTED** [review]
PRs should have a meaningful name like `Initial project setup`.
Also, every PR that changes UI should have a screenshot(s) and/or video.

File: QuizApp/AppDelegate.swift [hunk 6]
```diff
@@ -0,0 +1,23 @@
+
```
Comment (istaudacher) [inline QuizApp/AppDelegate.swift]: remove this line

File: QuizApp/SceneDelegate.swift [hunk 7]
```diff
@@ -0,0 +1,24 @@
+
```
Comment (istaudacher) [inline QuizApp/SceneDelegate.swift]: remove this line

File: QuizApp/ViewController.swift [hunk 8]
```diff
@@ -0,0 +1,13 @@
+
```
Comment (istaudacher) [inline QuizApp/ViewController.swift]: as above

**Review by istaudacher — CHANGES_REQUESTED** [review]

**Review by marinzg — APPROVED** [review]

**Review by istaudacher — APPROVED** [review]

**Pull request merged**